config/initializers/core_extensions/devise/strategies/json_web_token.rb

```diff
@@ -0,0 +1,27 @@
+# Your actual JWT Strategy
+# "config/initializers/core_extensions/devise/strategies/json_web_token.rb"
+
+module Devise
```

Comment:
Add reference in users model like `:bearer_token_authenticatable` and move this to the /lib folder.